core/src/tools/bsmtp.cc

```diff
@@ -276,31 +276,29 @@ int main(int argc, char* argv[])
 
   int mailport = default_port;
   std::string mailhost = default_mailhost;
+  std::string mail_and_port_source = "Mailhost and mailport set to default";
```

Comment:
What about `host_and_port_source` as variable name? I found it a bit less confusing.